lib/codegen.js

```diff
@@ -96,6 +100,7 @@ var getViewForSwagger2 = function(opts){
                 pathFormatString: path.replace(/{/g, "${parameters."),
                 className: opts.className,
                 methodName:  op.operationId ? normalizeName(op.operationId) : getPathToMethodName(opts, m, path),
+                version: getVersion(path),
```

Comment:
Please update the documentation as well to reflect that this is now available in the templates